### DISCUSSION TITLE
First power on of S-132, possible reset loop?

### DISCUSSION THREAD

**Discussion by mrthreeplates** [thehighnibble/thehighnibble.github.io#63] [S-132 add-on for IMSAI & Cromemco replicas]
Hi everyone, 

I've finished building my imsai8080 replica and it is working great.  I also have a S-132, however it doesn't seem to startup.  All I see on the vga are dots which print one at a time.  I'm guessing this might be some sort of reset loop.

The s-132 is supplying 5v power to the imsai, and the s-132 does the same thing whether or not it is standalone or attached. 

Pressing the resetvon the s-132 clears the screen and starts printing dots all over again.   I presume this isn't normal? 

My imsai has been flashed to the latest firmware. 

Any suggestions on what I should be seeing or how I might troubleshoot? 

I've cleaned the board, soldering looks good.

Thanks 

George

**Comment by thisspaceleftblank** [comment]
Did you add the initialization string to you boot.conf? (S132.init=1)

Charles

**Reply by mrthreeplates** [reply]
Thanks, no I didn't.   Let me try that.

**Reply by mrthreeplates** [reply]
That did the trick, thanks!

**Comment by nbreeden2** [comment]
Just so it's said the row of periods you saw on the S-132 VGA monitor were the S-132 looking for the IMSAI, not finding it then trying again.

**Reply by vbrownell** [reply]
I just finished my S-132 and thought I had a problem with the row of dots.  Thanks for posting the problem and solution!
